src/Api/RomeoShop.Backend.Api/Controllers/CatalogController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AutoMapper;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using RomeoShop.Backend.Contracts.Catalog;
+using RomeoShop.Backend.Cqrs.Commands.Product;
+using RomeoShop.Backend.Cqrs.Queries.Product;
+using RomeoShop.Backend.Dtos.Catalog;
+
+namespace RomeoShop.Backend.Api.Controllers
+{
+    [Route("api/v1/catalog")]
+    [ApiController]
+    public class CatalogController : Controller
```

Comment:
_Controller_ используется для MVC проектов. Для API - _ControllerBase_
```suggestion
    [Route("api/v1/catalog")]
    public class CatalogController : ControllerBase
```